packages/sdk/src/core/constants/chains.constant.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_RPC_URL = "https://rpc.gno.land:443"
```

Comment:
Please fix the lint.
- Please use single quotes instead of double quotes.
- Add semiclones.